hoot-core-test/src/test/cpp/hoot/core/io/OsmGeoJsonWriterTest.cpp

```diff
@@ -63,12 +63,16 @@ class OsmGeoJsonWriterTest : public HootTestFixture
 
   void runAllDataTypesTest()
   {
-    runTest("test-files/conflate/unified/AllDataTypesA.osm", "AllDataTypes.geojson");
+    Settings s;
+    s.set(ConfigOptions::getJsonFormatHootenannyKey(), false);
+    runTest("test-files/conflate/unified/AllDataTypesA.osm", "AllDataTypes.geojson", &s);
```

Comment:
These tests relied on the defaults that were set and don't accept the Hootenanny formatted JSON.